File: resources/[resources]/[standalone]/qpixel-lib/server/sv_sql.lua
SQL = SQL or {}

function Await(pPromise)
    return Citizen.Await(pPromise)
end

SQL.execute = function(query,param)
    local par
    if param == nil then
        par = {}
    else
        par = param
    end
    local psql = promise.new()
    exports.oxmysql:execute(query,par,function(result)
        psql:resolve(result)
    end)
    return psql
end

SQL.executedynamicparam = function(query, ...)
    local psql = promise.new()
    exports.oxmysql:execute(query,{...},function(result)
        psql:resolve(result)
    end)
    return psql
end

SQL.scalar = function(query, param)
    local par
    if param == nil then
        par = {}
    else
        par = param
    end
    local psql = promise.new()
    exports.oxmysql:scalar(query,par,function(result)
        psql:resolve(result)
    end)
    return psql
end

SQL.scalardynamicparam = function(query, ...)
    local psql = promise.new()
    exports.oxmysql:scalar(query,{...},function(result)
        psql:resolve(result)
    end)
    return psql
end

File: resources/[resources]/[standalone]/qpixel-lib/server/sv_sql.js
const SQL = {};

function prepareParams(params) {
    if (Array.isArray(params)) {
        return params;
    }
    const newObj = {};
    try {
        Object.keys(params).forEach((key) => {
            newObj[key] = params[key];
        });
    } catch (err) {}
    return newObj;
}

SQL.execute = (query, params) => {
    return new Promise((resolve, reject) => {
        exports.oxmysql.execute(query, prepareParams(params), (result, err) => {
            if (err) reject(err);
            resolve(result);
        });
    });
}

exports("getDate", () => {
    return Date.now()
});